README.md

```diff
@@ -272,10 +269,43 @@ const response = await replicate.predictions.create(options);
   "metrics": {},
   "created_at": "2022-04-26T22:13:06.224088Z",
   "started_at": null,
-  "completed_at": null
+  "completed_at": null,
+  "urls": {
+    "get": "https://api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq",
+    "cancel": "https://api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq/cancel",
+    "stream": "https://streaming.api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq"
+  }
 }
 ```
 
+#### Streaming
+
+Specify the `stream` option when creating a prediction
+to request a URL to receive streaming output using
+[server-sent events (SSE)](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events).
+
+If the requested model version supports streaming,
+then the returned prediction will have a `stream` entry in its `urls` property
+with a URL that you can use to construct an
+[`EventSource`](https://developer.mozilla.org/en-US/docs/Web/API/EventSource).
+
+```js
+if (prediction && prediction.urls && prediction.urls.stream) {
```

Comment:
```suggestion
if (prediction?.urls?.stream) {
```

Optional chaining has been a thing for a while, right?

Reply:
It should be, but I've seen too many cases of it not working as expected to trust it — even as recently as Node 18 with ESM.